### PR TITLE
Created about page and new class

### PR DESCRIPTION
**Pull request by jplemosx**
Segue !

### CODE REVIEW

File: assets/css/10ten.css [hunk 1]
```diff
@@ -7,7 +7,7 @@ main { min-height: 80vh; }
     align-items: center;
 }
 
-.mb-10 { margin-bottom: 10em; }
+.mb-10 { margin-top: 2em; margin-bottom: 10em; }
```
Comment (alexandremcosta) [inline assets/css/10ten.css]: create `.mt-2`

File: about.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                    Become an Instructor
+                </a>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+    <main class="flex-center">
+        <section class="text-center">
+            <h3 class="body-text">
```
Comment (alexandremcosta) [inline about.html]: unnecessary h3, move `body-text` to section

File: about.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                <a href="How it Works">How it Works</a>
+                <a href="become_instructor.html" class="btn btn-outline-primary">
+                    Become an Instructor
+                </a>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+    <main class="flex-center">
```
Comment (alexandremcosta) [inline about.html]: remove `flex-center` class

File: about.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        <section class="text-center">
```
Comment (alexandremcosta) [inline about.html]: remove `text-center`

File: assets/css/index.css [hunk 5]
```diff
@@ -11,4 +11,15 @@
     color: #444;
     margin-top: 1em;
     margin-bottom: 3em;
+}
+
+.index .body-text {
+    font-size: 1.5em;
+    font-weight: 300;
```
Comment (alexandremcosta) [inline assets/css/index.css:18]: when you remove `h3`, this font-weight will be unecessary

Comment (jplemosx) [reply assets/css/index.css:18]: After removing h3, I deleted the font-weight but the text was too fat without it, so I came back with the 300 font-weight and it looks better... should be the standard for regular body text in the website, pls take a look. Also, I believe the body-text left and right margins should be aligned with the elements in the menu (logo and become btn), maybe we need some adjustment there. Thx

File: about.html [hunk 6]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="assets/css/10ten.css" rel="stylesheet" />
+    <link href="assets/css/index.css" rel="stylesheet" />
```
Comment (alexandremcosta) [inline about.html:9]: should be about.css

File: assets/css/10ten.css [hunk 7]
```diff
@@ -8,6 +13,23 @@ main { min-height: 80vh; }
...
+    margin: 0 auto;
+}
+
+section {
+    padding: 10px 0;
+}
+
+.box {
+    padding: 1%;
+}
```
Comment (alexandremcosta) [inline assets/css/10ten.css:32]: Why this? Not sure if I understand...
I would add padding utility functions like we did for mt-5 and mt-2.
I would use p-1, p-2, just like bootstrap: https://getbootstrap.com/docs/5.0/utilities/spacing/

File: assets/css/10ten.css [hunk 8]
```diff
@@ -22,6 +44,94 @@ main { min-height: 80vh; }
...
+header {
+    background-image: linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7));
+    background-size: cover;
+    background-position: top;
+    width: 100%;
+    height: 65vh;
+    margin-top: 2em;
+    margin-bottom: 0;
+    
+}
```
Comment (alexandremcosta) [inline assets/css/10ten.css:115]: this will be applied to all 10ten headers?

File: howitworks.html [hunk 9]
```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="assets/css/10ten.css" rel="stylesheet" />
+    <link href="assets/css/index.css" rel="stylesheet" />
```
Comment (alexandremcosta) [inline howitworks.html:9]: howitworks.css

File: assets/css/10ten.css (hunk 7 above)
Comment (jplemosx) [reply assets/css/10ten.css:32]: I don't know why that entered, couldn't locate that new class so I just deleted it.

File: assets/css/10ten.css (hunk 8 above)
Comment (jplemosx) [reply assets/css/10ten.css:115]: Yep.. we are calling HEADER* the "Hero area" with the background image, just below the MENU, existent only in the HowItWorks and BecomeAnInstructor pages (there is no HEADER in the Index page for example..). There should be a minimum margin top for the text and a small margin between the button. 
(*) I don't know if this is right... what is the Header? Is it different from the Menu/Nav Bar ?

**Pull request merged**